pom.xml

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.facebook.airlift</groupId>
+        <artifactId>airbase</artifactId>
+        <version>98</version>
+    </parent>
+
+    <groupId>com.facebook.presto.yugabyte</groupId>
+    <artifactId>yugabyte-driver</artifactId>
+    <version>3.6.0-2-SNAPSHOT</version>
+
+    <name>cassandra-driver</name>
+    <description>Shaded version of Yugabyte Java Driver for Apache Cassandra</description>
+    <url>https://github.com/prestosql/presto-yugabyte-driver</url>
+
+    <licenses>
+        <license>
+            <name>Apache License 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <developers>
+        <!-- see original projects for developer information -->
+    </developers>
+
+    <scm>
+        <connection>scm:git:git://github.com/prestosql/presto-yugabyte-driver.git</connection>
+        <developerConnection>scm:git:git://github.com/prestosql/presto-yugabyte-driver.git</developerConnection>
+        <url>https://github.com/prestosql/presto-yugabyte-driver</url>
+        <tag>HEAD</tag>
+    </scm>
+
+    <properties>
+        <shadeBase>com.yugabyte.driver.\$internal</shadeBase>
```

Comment:
we should "put" code into our namespace:

io.prestosql.yugabyte.$internal

(is the backslash necessary?)

Reply:
I got the backslash from cassandra driver